### DISCUSSION TITLE
Will 7.2 be built for x86 linux?

### DISCUSSION THREAD

**Discussion by SpenceKonde** [avrdudes/avrdude#1407] [Q&A]
Need to know what to tell someone who is asking whether ATTinyCore 2.0.0 will support 32-bit x86 linux, and realized that if it's not being built, I would have to formally drop support for x86 Linux on all of my cores if I am to upgrade to 7.2 too

**Comment by mcuee** [comment]
Hmm. Where do you get the idea that 32bit Linux is not supported?

**Comment by mcuee** [comment]
Reference: avrdude project does not provide Linux or macOS binary
* https://github.com/avrdudes/avrdude/issues/987

avrdude project provides Windows x86/x64/ARM64 binaries for avrdude 7.1 release. Same for the upcoming 7.2 release.
https://github.com/avrdudes/avrdude/releases/tag/v7.1

Arduino avrdude-packaging project provides avrdude 7.1 binary, which include 32bit Linux x86 binary and 64bit Linux x64 binary. They only provide 32bit Windows binary. I am not so sure how fast they will provide the 7.2 release binaries.
https://github.com/arduino/avrdude-packing/releases/tag/7.1-arduino.1

**Comment by SpenceKonde** [comment]
That's good to to hear, though the fact that it won't be immediately available - I don't know what to do about that? Delay the core? Release the core only for some architectures pending release of the tools? What would you suggest? 

I absolutely get complaints whenever x86 linux is missing as it is in some avrdude builds from Arduino.

**Comment by mcuee** [comment]
@umbynos

The plan is to release avrdude 7.2 some time in July 2023. Just wondering what will be the time line for avrdude-packaging project to catch up with 7.2 release. I understand that it may take some time for testing.

**Reply by umbynos** [reply]
Hey @mcuee! We'll try to be as fast as possible.

**Comment by SpenceKonde** [comment]
Arduino tests stuff before they release it? Coulda fooled me.

____________
Spence Konde
Azzy’S Electronics

New products! Check them out at tindie.com/stores/DrAzzy
GitHub: github.com/SpenceKonde
ATTinyCore: Arduino support for almost every ATTiny microcontroller
Contact: ***@***.***

**Comment by mcuee** [comment]
@SpenceKonde 

In the mean time, you can give my test build a try, which is based on the same container and toolchain from Arduino avrdude-packing project. It has the binary build for Linux x86/x64/ARMv6/ARM64, macOS x64 and WIndows x86.
https://github.com/mcuee/avrdude-packing/releases/tag/snapshot_11Jun2023

**Comment by mcuee** [comment]
When avrdude 7.2 gets released, I will create un-official binaries like what I did for 7.1 release.
https://github.com/mcuee/avrdude/releases/tag/v7.1
